tests/PHPStan/Rules/Methods/MethodSignatureRuleTest.php

```diff
@@ -210,7 +210,15 @@ public function testBug3997(): void
 		$this->reportStatic = true;
 		$this->analyse([__DIR__ . '/data/bug-3997.php'], [
 			[
-				'Return type (string) of method Bug3997\Ipsum::count() should be compatible with return type (int) of method Countable::count()',
+				'Return type (int) of method Bug3997\Baz::count() should be covariant with return type (int<0, max>) of method Countable::count()',
```

Comment:
Oh, this is actually an annoying BC break :( Can you please do a similar thing that was done here https://github.com/phpstan/phpstan-doctrine/pull/290 and have the `Countable` stub be applied only for bleedingEdge? Putting `Countable` stub into a separate file called `Countable.stub` and configure it only in bleedingEdge.neon should be sufficient.

Reply:
should be fixed, I think.